bigframes/_config/bigquery_options.py

```diff
@@ -35,6 +35,12 @@
 
 UNKNOWN_LOCATION_MESSAGE = "The location '{location}' is set to an unknown value. Did you mean '{possibility}'?"
 
+LARGE_RESULTS_WARNING_MESSAGE = (
+    "Sampling is disabled because 'allow_large_results' is set to False. "
+    "To prevent downloading excessive data, it is recommended to limit the data "
+    "fetched with methods like .head() or .sample() before proceeding with downloads."
```

Comment:
Might be better to point folks at peek().

```suggestion
    "To prevent downloading excessive data, it is recommended to use the peek() method, "
    "or limit the data with methods like .head() or .sample() before proceeding with downloads."
```